codalab/worker/main.py

```diff
@@ -222,7 +222,7 @@ def parse_args():
     parser.add_argument(
         '--kubernetes-cert',
         type=str,
-        help='Contents of the SSL cert for the Kubernetes cluster. Only applicable if --bundle-runtime is set to kubernetes and --kubernetes-cert-path is set to /dev/null.',
+        help='Base64 encoded contents of the SSL cert for the Kubernetes cluster. Only applicable if --bundle-runtime is set to kubernetes and --kubernetes-cert-path is set to /dev/null.',
```

Comment:
Is there not a way to use a path or environment variables?

Reply:
This is set using an environment variable -- `CODALAB_WORKER_MANAGER_GPU_KUBERNETES_CERT` -- when starting up codalab_service.py.

We could use a path, but in that case we'd need to copy the cert file over to the worker pods on kubernetes when running them, which I think is a more complex solution.

Reply:
Will try this